apps/src/templates/EligibilityConfirmDialog.jsx

```diff
@@ -0,0 +1,102 @@
+/** @file Confirm Dialog for Maker Discount Codes */
+import React, {Component} from 'react';
+import i18n from "@cdo/locale";
+import BaseDialog from "./BaseDialog";
+import DialogFooter from "./teacherDashboard/DialogFooter";
+import Button from "./Button";
+
+const styles = {
+  subtitle: {
+    fontSize: 16,
+  },
+  signatureBox: {
+    height: 40,
+    width: 98 + '%',
+  }
+};
+
+export default class EligibilityConfirmDialog extends Component {
+  state = {
+    signature: "",
+    activeButton: false,
+  };
+
+  verifyResponse = () => {
+    if (this.check1.checked &&
+        this.check2.checked &&
+        this.check3.checked &&
+        this.state.signature !== ""){
```

Comment:
Nit: I'd require some non-whitespace content in the signature to pass this check.  Easy way to do that:
```js
/\S/.test(this.state.signature)
```